--- src/autogen-5.18.16/agen5/mk-stamps.sh.new	2020-06-19 12:40:35.000000000 -0400
+++ src/autogen-5.18.16/agen5/mk-stamps.sh	2020-06-19 12:17:31.000000000 -0400
@@ -28,7 +28,7 @@
 #  because some of the rules are complex and we don't want to
 #  deal with the dual update problem.
 
-STAMP_TEMP_DIR=$(mktemp -d /tmp/mk-stamps-XXXXXXXX)
+STAMP_TEMP_DIR=$(mktemp --suffix=.tdir -d /tmp/mk-stamps-XXXXXXXX)
 exec 9>&2 2>> ${STAMP_TEMP_DIR}/mk-stamps.log
 stop_tracing=:
 if (shopt -qo xtrace)
